pvcviewer-controller/rockcraft.yaml

```diff
@@ -0,0 +1,45 @@
+# Source https://github.com/kubeflow/kubeflow/blob/v1.9.0/components/pvcviewer-controller/Dockerfile
+name: pvcviewer-controller
+summary: An image for PVCViewer
+description: |
+  This image is used as part of the Charmed Kubeflow product. 
+  PVCViewers enable users to open a filebrowser on arbitrary persistent volume claims, letting them inspect, download, upload and manipulate data.
+version: "1.9.0"
+license: Apache-2.0
+base: ubuntu@22.04
+
+platforms:
+  amd64:
+
+services:
+  pvcviewer-controller:
+    override: merge
+    command: "pvc-viewer"
```

Comment:
Upstream command is `/manager` not pvc-viewer https://github.com/kubeflow/kubeflow/blob/v1.9.0/components/pvcviewer-controller/Dockerfile#L28